Documentation/Diagnostics/PH2111.md

```diff
@@ -0,0 +1,98 @@
+# PH2111: Reduce Cognitive Load
+
+| Property | Value  |
+|--|--|
+| Package | [Philips.CodeAnalysis.MaintainabilityAnalyzers](https://www.nuget.org/packages/Philips.CodeAnalysis.MaintainabilityAnalyzers) |
+| Diagnostic ID | PH2111 |
+| Category  | [Maintainability](../Maintainability.md) |
+| Analyzer | [ReduceCognitiveLoad](https://github.com/philips-software/roslyn-analyzers/blob/master/Philips.CodeAnalysis.MaintainabilityAnalyzers/Maintainability/ReduceCognitiveLoadAnalyzer.cs)
+| CodeFix  | No |
+| Severity | Error |
+| Enabled By Default | Yes |
+
+## Introduction
+
+Somewhat similiar to cyclomatic complexity, this analyzer calculates an overall complexity score for each method. The analyzer triggers if the score exceeds the maximum allowed value.
+
+The algorithm works as follows:
+* 1 point to start
+* 1 additional point for each instance of these tokens:
+  - `||`
+  - `&&`
+  - `!`
+  - `!=`
+  - `break`
+  - `continue` 
+* <I>n</I> additional points for each code block (i.e., each pair of `{...}`), where <I>n</I> is level of nesting involved.
+
+In practice, heavy nesting dominates the scoring.
```

Comment:
I would say "In most cases" as you don't know the style of programming that the reader is using